### ISSUE TITLE
Bug: licensecheck fails to identify all licenses of docutils package

### ISSUE THREAD

**Issue by NicolaDonelli** [FHPythonUtils/LicenseCheck#20]
I am publishing a python package, licensed with a MIT license, that depends on [docutils == 0.17.1](https://docutils.sourceforge.io/). According to its [license page](https://docutils.sourceforge.io/COPYING.html) docutils is licensed as Public Domain, Python Software, BSD2 and GPL but it seems that licenscecheck fails to correctly parse the GPL and BSD2.

# Reproducible example

OS: MacOS Monterey 12.6

Requirements file: requirements.txt
```
docutils==0.17.1
```

Project License: MIT

Command: `python -m licensecheck --using 'requirements:requirements.txt'`
Output: 
```
WARN: 'GNU GENERAL PUBLIC LICENSE (GPL)' License not identified so falling back to NO_LICENSE
┌──────┬────────────────────┬──────────────────────────────┐
│Compat│      Package       │           License            │
├──────┼────────────────────┼──────────────────────────────┤
│  ❌  │docutils            │Public Domain, Python Software│
└──────┴────────────────────┴──────────────────────────────┘
```

Expected Output: 
```
WARN: 'GNU GENERAL PUBLIC LICENSE (GPL)' License not identified so falling back to NO_LICENSE

┌──────┬────────────────────┬────────────────────────────────────────┐
│Compat│      Package       │           License                      │
├──────┼────────────────────┼────────────────────────────────────────┤
│  ❌  │docutils            │Public Domain, Python Software, GPL, BSD│
└──────┴────────────────────┴────────────────────────────────────────┘
```

**Comment by NicolaDonelli** [comment]
P.s. Sorry for having mistakenly labelled the question as a question: I realised after publishing it that it could probably be better categorised as a bug but it seems I cannot change the label anymore...

**Comment by FredHappyface** [comment]
Hi thanks for opening the issue. What is happening is that the terminal reporter/ formatter is cutting off the licenses to fit the table. Using --formatter markdown yields 

```py
python3 -m licensecheck --using 'requirements:requirements.txt;requirements_optional.txt' --format markdown > foo.md
```

![image](https://user-images.githubusercontent.com/41634689/197307772-0835d000-0d63-43b6-bbb5-db1099fd9fbe.png)

**Comment by FredHappyface** [comment]
Fixed in 2022.2 :)
